app/containers/Questions/Content/Pagination/Pagination.tsx

```diff
@@ -26,6 +26,19 @@ const Pagination: React.FC<PaginationProps> = ({
   setPage,
 }): JSX.Element => {
   const isDesktop450 = useMediaQuery('(min-width: 451px)');
+  const lastPage = page === totalPages;
+
+  const scrollToTop = () => {
+    setTimeout(
+      () =>
+        window.scrollTo({
+          top: 0,
+          left: 0,
```

Comment:
if its only scroll to top, u dont need left: 0,

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1419/commits/f89724b890eeac23863cf5691d27e1577c0f99d9